roles/kustomize_deploy/tasks/install_operators.yml

```diff
@@ -14,6 +14,43 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Generate values.yaml for OLM resources
+  vars:
+    cifmw_architecture_scenario: 'common/olm'
+    cifmw_ci_gen_kustomize_values_src_file: >-
+      {{
+        (
+          cifmw_kustomize_deploy_olm_source_files,
+          'values.yaml') | path_join
+      }}
+    cifmw_ci_gen_kustomize_values_userdata: >-
+      {{
+        (
+          cifmw_architecture_user_kustomize is defined and
```

Comment:
we'll need to check the doc to ensure this top-level param is properly exposed/documented - I'm not sure I did any good job on that field when I introduced it for the flexible loop.